crates/cli/src/subcommands/identity.rs

```diff
@@ -153,12 +142,7 @@ fn get_subcommands() -> Vec<Command> {
             .arg(Arg::new("identity").required(true).help(
                 "The identity you would like to recover. This identity must be associated with the email provided.",
             ).value_parser(clap::value_parser!(Identity)))
-            .arg(
-                Arg::new("server")
-                    .long("server")
-                    .short('s')
-                    .help("The server from which to request recovery codes"),
```

Comment:
note that this helptext changed by using the `common_args` version. I don't personally feel that this "custom" version is meaningfully more helpful.